### ISSUE TITLE
Error in documentation

### ISSUE THREAD

**Issue by Danita** [elasticinbox/elasticinbox-java#34]
From:
http://elasticinbox.github.io/api/v2/label/

> Returns all labels, their IDs, and optionally metadata (number of total and unread messages in the label).
> `PUT /rest/v2/:domain/:user/mailbox/purge`

Should be

> `GET /rest/v2/:domain/:user/mailbox/`

**Comment by Danita** [comment]
Also:

> Response without metadata:
> `"1334": "MyLabel^NestedLabel"`

Shouldn't it be...

> `"1334": "MyLabel/NestedLabel"`

...as per https://github.com/elasticinbox/elasticinbox/wiki/Labels-and-Markers ?

**Comment by rstml** [comment]
Thanks for reporting this.

Currently nested label separator is `^` symbol. (see https://github.com/elasticinbox/elasticinbox/blob/master/modules/core/src/main/java/com/elasticinbox/core/model/LabelConstants.java#L45)

In early days we decided to switch from `/` to `^` because forward slash if often used for label names (e.g. IMAP folders). Therefore we wanted to avoid clash and use symbol which is used rarely.

It's important to note that currently there's no use of nested labels internally. Support was added as a placeholder for future IMAP implementation.

I've updated Labels and Markers page and will fix API docs shortly.
